src/custom-components/Pagination/usePagination.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect } from "react";
+
+export const DOTS = "...";
+
+interface usePaginationInterface {
+  totalCount: number;
+  pageSize: number;
+  currentPage: any;
+}
+
+const range = (start: number, end: number) => {
+  let length = end - start + 1;
+  return Array.from({ length }, (_, idx) => idx + start);
+};
+
+var paginationRange: any[] = [];
```

Comment:
Oh, based on this [Junior's comment](https://github.com/FromDoppler/doppler-sms-webapp/pull/4/files#r1055579737) I understand why you have used `any` type.

You could use a union like this: 

```suggestion
var paginationRange: (number | DOTS)[] = [];
```

(or something similar)

Reply:
I saw that it is done 👍. Resolving the conversation...